### DISCUSSION TITLE
Build Ubuntu 22.04 LTS and 24.04 LTS #460 and #462

### DISCUSSION THREAD

**Discussion by wmport** [Joshua-Riek/ubuntu-rockchip#770] [General]
Build Ubuntu 22.04 LTS and 24.04 LTS [#460](https://github.com/Joshua-Riek/ubuntu-rockchip/actions/runs/8928473525)
The installer does not apply the changes. Nanopc-t6. desktop version.
[oem-config.log](https://github.com/Joshua-Riek/ubuntu-rockchip/files/15204936/oem-config.log)

**Comment by Joshua-Riek** [comment]
What do you mean? You cant install a desktop on both Ubuntu 24.04 and 22.04?

**Reply by wmport** [reply]
only 24.04 installer does not apply the changes
I haven't tried version 22.04
The installer does not apply the changes, there is an animation of the application window, no changes, just wallpaper and that’s it

**Reply by wmport** [reply]
in build [462](https://github.com/Joshua-Riek/ubuntu-rockchip/actions/runs/8944098899) the installer also does not work, there is no “changes are applied” animation, no GUI restart
nanopc-t6 .. 24.04

**Reply by wmport** [reply]
Video confirmation

https://github.com/Joshua-Riek/ubuntu-rockchip/assets/159494449/f4713e45-1ade-4487-97df-bd62f6cb964d

**Reply by Joshua-Riek** [reply]
there seems to be a dbus issue causing the program to crash, I'm debugging now.

**Reply by Joshua-Riek** [reply]
It's fixed for now, for some reason when building the rootfs with docker this will happen, and I have no idea why. I'll look deeper into it later.

Thanks for letting me know about this.
